### ISSUE TITLE
Regarding the loss function curve

### ISSUE THREAD

**Issue by LyuQi127** [Zhangwenyao1/DreamVLA#5]
Hi, I am reimplementing your work. How does the loss function change when only predicting trajectories and images？

**Comment by Zhangwenyao1** [comment]
If you want to solely predict the dynamic regions， you should finetune models with configs _--flow_as_mask_,  which means use flow as mask rather than predict the whole image and flow.
